src/index.html

```diff
@@ -7,7 +7,54 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0" >
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+
+    <div class="stars stars--1" >
+      <div class="stars__star stars__star--active" ></div>
```

Comment:
need to use nth-child to set active stars
```suggestion
      <div class="stars__star" ></div>
```